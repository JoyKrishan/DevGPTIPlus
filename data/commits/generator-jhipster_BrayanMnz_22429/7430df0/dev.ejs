      "keycloakVersion": "21.1.1",