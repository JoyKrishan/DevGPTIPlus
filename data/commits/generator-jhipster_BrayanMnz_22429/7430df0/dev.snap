            "https://localhost:8080/*",
            "https://localhost:8080",
      "keycloakVersion": "21.1.1",
            "https://localhost:8080/*",
            "https://localhost:8080",
      "keycloakVersion": "21.1.1",